playbooks/bond.yml

```diff
@@ -2,6 +2,12 @@
 - hosts: all
   become: True
   tasks:
+  # 'port' is only supported by nmstate 0.4 or later version
+    - name: Install nmstate package
+      package:
+        name: nmstate
+        state: present
```

Comment:
IMO, it would be nice to use the playbook introduced in #25

Reply:
That playbook is too specific in my opinion. Ideally, users should not need to care about Copr but be able to use the package from their distribution.

Reply:
I understand in that case, it looks good